android/build.gradle

```diff
@@ -23,21 +23,26 @@ apply plugin: 'com.android.library'
 
 dependencies {
     implementation 'io.ably:ably-android:1.1.10'
+    coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:1.1.1'
 }
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 30
 
     // Require Java language level 8 so we can use Method References (used with Lambdas)
     compileOptions {
-        sourceCompatibility 1.8
-        targetCompatibility 1.8
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+        // Flag to enable Java 8 language APIs while supporting to older SDK Versions
```

Comment:
Please insert a blank line above this comment to make it clearer that it lives with the line below.

Reply:
Addressed: a23b04ef17039c4a68d05220ea76366e3e0e974e